package.json

```diff
@@ -34,7 +34,7 @@
     "@angular-devkit/build-ng-packagr": "~0.13.0",
     "@commitlint/cli": "^8.3.5",
     "@commitlint/config-conventional": "^8.3.4",
-    "@compodoc/compodoc": "1.1.8",
+    "@compodoc/compodoc": "~1.1.11",
```

Comment:
We cannot just upgrade `compodoc` version because the code blocks are not rendered correctly anymore since version 1.1.9. That is why [we pinned the version to 1.1.8 some time ago](https://github.com/NationalBankBelgium/stark/pull/1198).

I think it is better to create a separate PR with this upgrade because some JSDocs will have to be adapted so that they are correctly displayed in the latest version of compodoc. For example the docs of the `StarkProgressIndicatorDirective`:

Before:
<img width="721" alt="Screenshot 2020-01-29 at 16 02 41" src="https://user-images.githubusercontent.com/5698318/73368196-deed6a80-42b0-11ea-8dbf-0592f1bec136.png">


And now: 
<img width="721" alt="Screenshot 2020-01-29 at 16 02 53" src="https://user-images.githubusercontent.com/5698318/73368219-e57be200-42b0-11ea-8e25-55f5d1224261.png">

Reply:
The issue about the code blocks I mentioned is only for those blocks that we have in our JSDocs like this. In the StarkProgressIndicator directive we have:

```
/**
 * This directive must be used as attribute on a DOM element and the config provided should be a {@link StarkProgressIndicatorConfig} object:
 *
 * ```html
 * <some-element [starkProgressIndicator]="{topic: 'some-topic', type: 'SPINNER'}" ></some-element>
 * <!-- or -->
 * <some-element [starkProgressIndicator]="progressIndicatorConfig" ></some-element>
 * ```
 *
 * ...
*/
```

In order to display it correctly in the latest version of compodoc, the `*` at the begining of each line of the code block should be removed:

```
/**
 * This directive must be used as attribute on a DOM element and the config provided should be a {@link StarkProgressIndicatorConfig} object:
 *
  ```html
  <some-element [starkProgressIndicator]="{topic: 'some-topic', type: 'SPINNER'}" ></some-element>
  <!-- or -->
  <some-element [starkProgressIndicator]="progressIndicatorConfig" ></some-element>
  ``'
 *
 * ...
*/
```

But then, TSLint will complain due to the `jsdoc-format` rule, so it should be disabled in those cases 😅